test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/Helpers/EnvironmentHelper.cs

```diff
@@ -330,5 +330,7 @@ private void SetDefaultEnvironmentVariables()
         CustomEnvironmentVariables["OTEL_TRACES_EXPORTER"] = "none";
         CustomEnvironmentVariables["OTEL_METRICS_EXPORTER"] = "none";
         CustomEnvironmentVariables["OTEL_LOGS_EXPORTER"] = "none";
+
+        CustomEnvironmentVariables["OTEL_DOTNET_AUTO_PLUGINS"] = "Splunk.OpenTelemetry.AutoInstrumentation.Plugin, Splunk.OpenTelemetry.AutoInstrumentation, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null";
```

Comment:
Should we simplify it just to?:
`"Splunk.OpenTelemetry.AutoInstrumentation.Plugin, Splunk.OpenTelemetry.AutoInstrumentation";`

Reply:
It works for me. Thanks!